### DISCUSSION TITLE
Multiple Ideas (Because I realized how many add-on ideas I had in my head)

### DISCUSSION THREAD

**Discussion by SpryzenR** [ScratchAddons/ScratchAddons#6141] [Ideas] [answered]
_**SpryzenRYT Addon Ideas:**_

**A new feature for the Scratch Addons Pop-up Extension:**
If you copied and pasted a scratch link you got from your friend or a very popular project like griffpatch's 2D Minecraft Project you can find out the stats for the project, like how many comments, remixes, etc.

This might be useless seeing as you'd have to actually LOOK at the project, copy and paste its link and keep it in your clipboard or something, but the stats are right there for you anyways but that's just another thing I thought of. 

**Addons for Editor:**
Variable manger add-on: make an option to hide the variable stat (like a check mark or something so you show the stat again if you hid it)

For a new add-on make like something to select multiple scripts. Then you can be given the option to either move or delete multiple scripts. 

Hide Enlarge/Shrink/Equalize?? Buttons (idk what they're called but you can find them on the bottom right of the code area.) Make a button that hide those buttons (idk what they're called) on the bottom right of the code area. When hidden make a button that shows them. For example a drop down/up menu or something.

**Addons for My Stuff:**
In the my stuff there is a sort by button. Another add-on I thought of was to organize it by share order. Like there could two options, one is sort by Last Shared (Newest to Oldest) and the other is sort by Last Shared (Oldest to Newest) 

--------------------------------------------------------------------------------------------------------------------------------------------------------
I'm @SpryzenRYT on scratch.mit.edu, please consider giving me a follow and check in on what I'm doing 😄

**Comment by DNin01** [comment, marked as answer]
> **A new feature for the Scratch Addons Pop-up Extension:**
> 
> If you copied and pasted a scratch link you got from your friend or a very popular project like griffpatch's 2D Minecraft Project you can find out the stats for the project, like how many comments, remixes, etc.
> 
> 
> 
> This might be useless seeing as you'd have to actually LOOK at the project, copy and paste its link and keep it in your clipboard or something, but the stats are right there for you anyways but that's just another thing I thought of. 

An alternative would be displaying those statistics on the actual project page instead of in the extension popup.

> 
> **Addons for Editor:**
> 
> Hide Enlarge/Shrink/Equalize?? Buttons (idk what they're called but you can find them on the bottom right of the code area.) Make a button that hide those buttons (idk what they're called) on the bottom right of the code area. When hidden make a button that shows them. For example a drop down/up menu or something.

We do have a very similar feature, but it's tucked away in the Customizable zoom controls addon or whatever it's called.

> **Addons for My Stuff:**
> 
> In the my stuff there is a sort by button. Another add-on I thought of was to organize it by share order. Like there could two options, one is sort by Last Shared (Newest to Oldest) and the other is sort by Last Shared (Oldest to Newest) 

If you go to your profile page and click "View all" next to the Shared Projects row, you get a list sorted by last shared. Even if we decided to make a sort by last shared feature for the My Stuff page, it would likely not run very quickly because we'd have to load the entire list before we can properly sort it.
